src/gax-internal/src/observability/attributes.rs

```diff
@@ -18,37 +18,53 @@
 /// Span Kind for OpenTelemetry interop.
 ///
 /// Always "Client" for a span representing an outbound HTTP request.
-pub const KEY_OTEL_KIND: &str = "otel.kind";
+pub(crate) const KEY_OTEL_KIND: &str = "otel.kind";
 /// Span Name for OpenTelemetry interop.
 ///
 /// If `url.template` is available use "{http.request.method} {url.template}", otherwise use "{http.request.method}".
-pub const KEY_OTEL_NAME: &str = "otel.name";
+pub(crate) const KEY_OTEL_NAME: &str = "otel.name";
 /// Span Status for OpenTelemetry interop.
 ///
 /// Use "Error" for unrecoverable errors like network issues or 5xx status codes.
 /// Otherwise, leave "Unset" (including for 4xx codes on CLIENT spans).
-pub const KEY_OTEL_STATUS: &str = "otel.status";
+pub(crate) const KEY_OTEL_STATUS: &str = "otel.status";
 
 /// The string representation of the gRPC status code.
-pub const KEY_GRPC_STATUS: &str = "grpc.status";
+pub(crate) const KEY_GRPC_STATUS: &str = "grpc.status";
 
 // Custom GCP Attributes
 /// The Google Cloud service name.
 ///
 /// Examples: appengine, run, firestore
-pub const KEY_GCP_CLIENT_SERVICE: &str = "gcp.client.service";
+pub(crate) const KEY_GCP_CLIENT_SERVICE: &str = "gcp.client.service";
 /// The client library version.
 ///
 /// Example: v1.0.2
-pub const KEY_GCP_CLIENT_VERSION: &str = "gcp.client.version";
+pub(crate) const KEY_GCP_CLIENT_VERSION: &str = "gcp.client.version";
 /// The client library repository.
 ///
 /// Always "googleapis/google-cloud-rust".
-pub const KEY_GCP_CLIENT_REPO: &str = "gcp.client.repo";
+pub(crate) const KEY_GCP_CLIENT_REPO: &str = "gcp.client.repo";
 /// The client library crate name.
 ///
 /// Example: google-cloud-storage
-pub const KEY_GCP_CLIENT_ARTIFACT: &str = "gcp.client.artifact";
+pub(crate) const KEY_GCP_CLIENT_ARTIFACT: &str = "gcp.client.artifact";
+
+// Error Type Constants
+/// A client-configured timeout was reached.
+pub const ERROR_TYPE_CLIENT_TIMEOUT: &str = "CLIENT_TIMEOUT";
```

Comment:
Optional: consider refactoring these under a module, the names are a bit repetitive.